payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentResult.kt

```diff
@@ -18,6 +18,9 @@ internal sealed class PaymentResult : Parcelable {
     @Parcelize
     object Canceled : PaymentResult()
 
+    @Parcelize
+    object TimedOut : PaymentResult()
```

Comment:
I think timeout can be represented as a failure. I don't think it needs its own constant.